src/js/modules/supervisors/FileManager.ts

```diff
@@ -157,16 +157,16 @@ class FileManager {
       );
       return Promise.resolve();
     } else {
+      this.repository.remove(handle);
```

Comment:
What if this fails?

Reply:
why does it fail? `handle` is a valid entry